packages/server/graphql/types/User.ts

```diff
@@ -50,7 +50,6 @@ const User = new GraphQLObjectType<any, GQLContext, any>({
       description: 'The userId provided by us'
     },
     allAvailableIntegrations: require('../queries/allAvailableIntegrations').default,
-    archivedTasks: require('../queries/archivedTasks').default,
```

Comment:
let's bring this back & just flag it as deprecated for a version or 2.
if a user hasn't hit refresh yet & requests archivedTasks, we don't want it to break for them.
so stage 1 we change what the client requests.
then in stage 2 when most of the clients are requesting the new thing, we can remove the deprecated old thing